OracleManagementAgent/LICENSE.txt

```diff
@@ -0,0 +1,34 @@
+The Universal Permissive License (UPL), Version 1.0
```

Comment:
This file is not required. We already have a `LICENSE.txt` in this repo.

Reply:
removed